supply/lib/supply/options.rb

```diff
@@ -121,7 +121,13 @@ def self.available_options
                                      verify_block: proc do |value|
                                        available = valid_tracks
                                        UI.user_error! "Invalid value '#{value}', must be #{available.join(', ')}" unless available.include? value
-                                     end)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :validate_only,
+                                     env_name: "SUPPLY_VALIDATE_ONLY",
+                                     optional: true,
+                                     description: "If true, changes will only be validated with Google Play rather than actually uploaded",
```

Comment:
Thanks for the review! Updated the comment per your suggestion.

As I understand the validate functionality, it uploads the data but doesn't change anything. So if you don't call the commit function, nothing about your listing will change. (See the documentation here: https://developers.google.com/android-publisher/api-ref/edits/validate) So I think with the new wording, we're reasonably accurate.